packages/chatbot-server-mongodb-public/src/verifiedAnswers.eval.ts

```diff
@@ -0,0 +1,291 @@
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "reset my password",
+    similarVerifiedAnswerQuery: "Can i reset my password",
+    verifiedAnswerIndex,
+    tags: ["perturbation", "iam", "should_match"],
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "reset database password",
+    similarVerifiedAnswerQuery: "Can i reset my password",
+    verifiedAnswerIndex,
+    tags: ["perturbation", "iam", "should_match"],
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "connect to stream process",
+    verifiedAnswerIndex,
+    tags: ["atlas_stream_processing", "should_not_match"],
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "connect to database kotlin",
+    verifiedAnswerIndex,
+    tags: ["driver", "kotlin", "should_not_match"],
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "connect to database with Kotlin coroutine driver",
+    verifiedAnswerIndex,
+    tags: ["driver", "kotlin", "kotlin_coroutine_driver", "should_not_match"],
+  }),
+  // 👇 From EAI-580 👇
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "how do I set up billing alerts in Atlas",
+    // No similar verified answer
+    tags: ["billing", "should_not_match"],
+    verifiedAnswerIndex,
+  }),
+  makeVerifiedAnswerEvalCase({
+    inputQuery: "how do I set up billing alerts in Atlas?",
+    // No similar verified answer
+    tags: ["billing", "should_not_match"],
+    verifiedAnswerIndex,
+  }),
+];
+
+// Helper function to create a verified answer eval case
+function makeVerifiedAnswerEvalCase(args: {
+  inputQuery: string;
+  similarVerifiedAnswerQuery?: string;
+  description?: string;
+  tags?: MongoDbVerifiedAnswerTag[];
+  verifiedAnswerIndex: VerifiedAnswerIndex;
+}): VerifiedAnswersEvalCase {
+  return {
+    input: {
+      query: args.inputQuery,
+    },
+    expected: {
+      answer: args.similarVerifiedAnswerQuery
+        ? findExactVerifiedAnswer(
+            args.similarVerifiedAnswerQuery,
+            args.verifiedAnswerIndex
+          )
+        : undefined,
+    },
+    tags: args.tags,
+    metadata: {
+      similarVerifiedAnswerQuery: args.similarVerifiedAnswerQuery,
+      description: args.description,
+    },
+  };
+}
+
+// -- Evaluation metrics --
+const MatchesSomeVerifiedAnswer: VerifiedAnswersEvalCaseScorer = (args) => {
+  return {
+    name: "MatchesSomeVerifiedAnswer",
+    score: args.output.answer ? 1 : 0,
+  };
+};
+
+const MatchesExpectedOutput: VerifiedAnswersEvalCaseScorer = (args) => {
+  const isMatch = args.output.answer?.answer === args.expected.answer;
+  let matchType = "";
+  if (isMatch && args.expected.answer !== undefined) {
+    matchType = "true_positive";
+  } else if (isMatch && args.expected.answer === undefined) {
+    matchType = "true_negative";
+  } else if (!isMatch && args.expected.answer !== undefined) {
+    matchType = "false_positive";
+  } else if (!isMatch && args.expected.answer === undefined) {
+    matchType = "false_negative";
+  }
```

Comment:
Could be a little more concise here

```suggestion
  const matchType = [
    isMatch ? "true" : "false",
    args.expected.answer === undefined ? "negative" : "positive",
  ].join("_");
```